### DISCUSSION TITLE
Ramdisk for the HLS data ?

### DISCUSSION THREAD

**Discussion by FilipStadler** [owncast/owncast#1801] [Ideas]
I was thinking if I make a ramdisk for the HLS data to avoid usning the ssd/harddisk to much and maybe even improve speed how big should it be ?

**Comment by gabek** [comment]
It depends on how you have Owncast configured.

I suggest configuring Owncast how you like, then run it, look how much space it used, and then there's your answer.

**Reply by FilipStadler** [reply]
OK... I cant because I would like to have the HLS data on a ramdisk on a raspberry you can avoid to much traffic on a sd-card to wear it out also but also it makes sense to use a ramdisk for the hls if possible.

sudo mount -t tmpfs -o size=1g tmpfs /home/pi/owncast/data/hls

And then sudo systemctl start owncast but it have problems with permissions 

Anyway to pass parameters to the owncast about directory locations maybe ?

**Reply by gabek** [reply]
Regardless of where your ramdisk is located you'll have to fix the permissions for it. Make sure the user that is running Owncast can also write to your ramdisk.
